charts/llama-stack/files/run.yaml

```diff
@@ -0,0 +1,123 @@
+version: '2'
```

Comment:
How about making this a symlink of the vllm-gpu template file?

Reply:
I don't think that works because we would need to know that that file exists and where it exists on the user's machine.

And Helm can't access files outside of its own folder so not sure that works even if we did know where that file was. 

Unless I am misunderstanding the request.

Reply:
I see thanks for the clarification, maybe this could help alleviate my concern https://github.com/meta-llama/llama-stack/pull/1680/files#r2003272702?